scarf/assay.py

```diff
@@ -57,10 +56,10 @@ class Assay:
         cells:
         nthreads:
         rawData:
-        feats:
+        feats: a dataframe with info about each feature in the dataset
```

Comment:
Technically `feats` is not a dataframe. It is a Metadata object.

Reply:
True! I realised that for attr cells in other places, will change this here as well